src/js/modules/TileGroupSet.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import TileGroup from 'js/modules/TileGroup';
+
+
+const TileGroupSet = ({ groups, tileKey, groupTitleSubPath }) =>  {
+
+  // gets the number of topics that have active services
+  const countOfGroups = groups.edges
+    .map((g) => g.node.services.edges.length > 0)
+    .reduce((total, amount) => amount ? total + 1 : total, 0);
+  const shouldTilesStackVertical = countOfGroups > 1;
```

Comment:
This whole section that provides the logic for if we display the tiles stacked or side-by-side felt hacky but I couldn't figure out anything more elegant. As designed, it depends on whether the Theme has more than one Topic with active Service pages.  This gets passed into the TileGroup as the `direction` props which will be either `'column'` or `false`. v open to suggestions on how to make this better.

Reply:
This is gonna take me a minute to wrap my head around. and I don't wanna hold up moving forward and merging. Lemme pull this locally and review and make an issue if I can come up with something else.